cernopendata/modules/fixtures/data/docs/cod-portal-upgrade-31-1-2024/cod-portal-upgrade-31-1-2024.md

```diff
@@ -0,0 +1,14 @@
+A new version of the CERN Open Data Repository has been deployed in January 2024.
+Some of the highlights of this release are:
+
+* Change of technologies, from Angular to [ReactJS](https://react.dev/) and from Bootstrap to [Semantic UI](https://semantic-ui.com/)
+for the web interface, which improves the navigation and overall use experience.
+* Changes in the search functionality, including modifying the default search to do an `AND` of the terms (instead of `OR`),
+escaping the special character `/` (allowing searches by DOI).
+* Redesign facets for results filtering.
+* Change the self-maintained ElasticSearch instance, which had reached its end of life support, by the open-sourced
```

Comment:
I don't think it is necessary to mention. We could also say that we changed from Python 2.7 to Python 3.6 due to end-olf-life etc.  These kind of news are internal not relevant to the open data users because they don't see them.